config.js

```diff
@@ -0,0 +1,19 @@
+const config = {
+    template: "./src/index.hbs",
+    filename: "./index.html",
+    mobile: true,
+    site: {
+      title: "", // Add site title here
```

Comment:
Let's just use placeholder text instead of comments here. "Example Page Title", etc.